css-canvas-medical/styles.css

```diff
@@ -0,0 +1,165 @@
+.container{
+  max-width: auto;
+  margin: auto;
+}
+
+body{
+  font-family: "Tiempos Headline";
+}
+
+.row{
+  display: flex;
+}
+
+.column-full{
+  flex-basis: 100%;
+}
+
+.column-one-half{
+  flex-basis: 50%;
+}
+
+.column-one-fourth{
+  flex-basis: 25%;
+}
+
+.justify-center{
+  justify-content: center;
+}
+
+.footer h3{
+  font-family: Avenir;
+}
+
+h2{
+  color: rgb(92, 92, 92);
+  font-weight: 400;
+  font-family: "Tiempos Headline"
+}
+
+h3{
+  color: rgb(92, 92, 92);
+  font-weight: 400;
+  font-family: Avenir;
+}
+
+p{
+  color: rgb(92, 92, 92);
+  font-weight: 100;
+  font-family: Avenir;
+}
+
+.text-center{
+  text-align: center;
+}
+
+.gray-background{
+  padding: 30px;
+  background-color: rgb(245, 243, 243);
+}
+
+.h3{
+margin-bottom: 80px;
+}
+
+.row button{
+  font-weight: 100;
+  font-size: 20px;
+  margin-top: 30px;
+  padding: 10px 30px 10px 30px;
+  font-family: Avenir;
+  border-radius: 5px;
+  border-style: none;
+  background-color: #294465;
+  color: white;
+}
+
+.black-background{
+  background-color: rgb(85, 83, 83);
+}
+
+.white-text h1{
+  font-weight: 500;
+  color: rgb(247, 245, 245);
+}
+
+.white-text h3{
+  font-family: Avenir;
+  color: rgb(211, 211, 211);
+}
+
+.white-text p{
+  line-height: 1.5;
+  font-family: Avenir;
+  color: white;
+}
+
+.white-logo{
+  margin-right: 100px;
+  padding-left: 10px;
+  padding-right: 20px;
+  width: 20%;
+}
+
+.margin-top{
+  margin-top: 80px;
+}
+
+.address{
+  padding-left: 50px;
+  margin-left: 27%;
```

Comment:
**_Woah_**, that's a lot of margin. Why is this necessary? There is likely a better way to accomplish this.